ExportScripts/export_standalone_monolithic.py

```diff
@@ -222,14 +227,29 @@ def process_command(args: list,
 
     #commands are based on 
     #https://github.com/o3de/o3de-multiplayersample/blob/development/Documentation/PackedAssetBuilds.md
-
+    selected_platform = args.platform
+    if not selected_platform:
+        logger.info("Platform not specified! Defaulting to Host platform...")
+        system_platform = platform.system()
+        if not system_platform:
+            logger.error("Unable to identify host platform! Please supply the platform using '--platform'. Options are [pc, linux].")
+            sys.exit(1)
+        if system_platform == "Windows":
+            selected_platform = "pc"
+        elif system_platform == "Linux":
+            selected_platform = "linux"
+        else:
+            if system_platform == "Darwin":
+                system_platform = "Mac OS"
```

Comment:
The selected platform is set here for mac.
It should be "mac".

Also the selected platform corresponds to the ["asset" platform](https://github.com/o3de/o3de/blob/7566dffd3a6f5ccc7185c2c172fc755952058ce2/Code/Framework/AzCore/AzCore/PlatformId/PlatformDefaults.cpp#L59-L97) where "pc" can be seen as the albatross out of the options(it should be "windows", but backwards compatiblity makes that hard to change.)